tests/helpers/rmq/RabbitMQ.test.ts

```diff
@@ -14,6 +14,81 @@ describe('RabbitMQ', () => {
     mockAmqplibConnect.mockReset();
   });
 
+it('should re-bind consumer when connection is re-established', async () => {
```

Comment:
It would be great if we do tests on a real RMQ. We can launch the RMQ docker container for test on Github
This will allow us to perform all RMQ tests on a more realistic environment